lib/adagrams.rb

```diff
@@ -0,0 +1,166 @@
+# Chantelle and Semret (Pair-Programing)
+# Adagram project
+# 8/13/18
+require 'csv'
+
+# WAVE 1 SYNTAX
+def draw_letters
+
+  random_draw = []
+
+  letter_pool = ('A'..'Z').map do |char|
+    case char
+    when 'A', 'I'
+      letter = char * 9
+    when 'B', 'C', 'F', 'H', 'M', 'P', 'V', 'W', 'Y'
+      letter = char * 2
+    when 'D', 'L', 'S', 'U'
+      letter = char * 4
+    when 'E'
+      letter = char * 12
+    when 'G'
+      letter = char * 3
+    when 'J', 'K', 'Q', 'X', 'Z'
+      letter = char * 1
+    when 'N', 'R', 'T'
+      letter = char * 6
+    when 'O'
+      letter = char * 8
+    end
+  end
+
+  split_letter_pool = letter_pool.map do |chars|
+    chars.split("")
+  end
+
+  #random draw of 10 from array and flatten the letter pool
+  random_draw = split_letter_pool.flatten.sample(10)
+  return random_draw
+
+end
```

Comment:
I think overall, your solution to this method is good-- it shows that you all know how to use iteration and specifically `map` to create an interesting array. My only concern is that I feel like maybe `map` forced you all to need to make a lot of arrays and a lot of loops-- I don't recommend any changes at this moment, but if you all ever take a minute to look at this problem with fresh eyes, there may be other approaches where it doesn't feel like "fighting" with `map`